app/actions/views/channel.js

```diff
@@ -155,6 +155,26 @@ export function loadPostsIfNecessary(channelId) {
     };
 }
 
+export function loadPostsIfNecessaryWithRetry(channelId) {
```

Comment:
Do we need to have both `loadPostsIfNecessary...` functions any more? I can't think of anywhere that we'd want the plain `loadPostsIfNecessary` without the retry